src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$box-size: 100px;
+$margin-left: 103px;
```

Comment:
The variable name `$margin-left` should be more abstract and reusable across the project. Consider using a name that reflects its purpose or scope, such as `$gutter-width` or `$spacing-unit`, in accordance with the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).